### DISCUSSION TITLE
bash命令是否有误

### DISCUSSION THREAD

**Discussion by chenkang404** [zai-org/ChatGLM3#1192] [Q&A] [answered]
cd finetune_demo
OMP_NUM_THREADS=1 torchrun --standalone --nnodes=1 --nproc_per_node=8  finetune_hf.py  data/AdvertiseGen/  THUDM/chatglm3-6b  configs/lora.yaml configs/ds_zero_2.json
上述命令和finerune_hf.py这中的代码是否不一致：
@app.command()
def main(
        data_dir: Annotated[str, typer.Argument(help='')],
        model_dir: Annotated[
            str,
            typer.Argument(
                help='A string that specifies the model id of a pretrained model configuration hosted on huggingface.co, or a path to a directory containing a model configuration file.'
            ),
        ],
        config_file: Annotated[str, typer.Argument(help='')],
        auto_resume_from_checkpoint: str = typer.Argument(
            default='',
            help='If entered as yes, automatically use the latest save checkpoint. If it is a numerical example 12 15, use the corresponding save checkpoint. If the input is no, restart training'
        ),

):

**Comment by zRzRzRzRzRzRzR** [comment, marked as answer]
有啥不一样呢

auto_resume_from_checkpoint 不是必须的

**Reply by chenkang404** [reply]
代码里参数为四个：data_dir，model_dir, config_file, auto_resume_from_checkpoint; 
命令：OMP_NUM_THREADS=1 torchrun --standalone --nnodes=1 --nproc_per_node=8 finetune_hf.py data/AdvertiseGen/ THUDM/chatglm3-6b configs/lora.yaml configs/ds_zero_2.json 中，最后的两个configs/lora.yaml configs/ds_zero_2.json 都赋值给了config_file吗
